app/src/main/java/com/eveningoutpost/dexdrip/LibreReceiver.java

```diff
@@ -87,6 +87,10 @@ public void run() {
                                     processValues(currentRawValue, smoothingValues, context);
                                 }
                                 currentRawValue.save();
+                                // Libre2 Raw readings are also sent to Sync+ Followers if
+                                // show raw graph for Libre2 option is activated.
+                                if (Pref.getBoolean("plus_follow_master", false) && prefs.getBoolean("Libre2_showRawGraph",false))
+                                    GcmActivity.syncLibre2RawReading (currentRawValue);
```

Comment:
Please use consistent code formatting (no spaces before bracket of function parameter list), at least for the code you have changed.